### PR TITLE
Used express-validator to validate data in the backend.

### PR DESCRIPTION
**Pull request by MiguelCanedo**
Validated the following POST routes:
- create-brand
- create-product
- create-variant

### CODE REVIEW

File: backend/src/middleware/api/router.js [hunk 1]
```diff
@@ -35,46 +36,69 @@ router.get("/products", async (req, res) => {
...
-    console.log("\nCreating product with the following information:");
-    console.log(`${req.body}\n`);
+router.post("/create-product", 
+    validator.body("brandName").notEmpty().escape(),
+    validator.body("brandCategory").notEmpty().escape(),
+    validator.body("variantName").notEmpty().escape(),
+    validator.body("name").notEmpty().escape(),
+    validator.body("price").notEmpty().isInt(),
+    validator.body("quantity").notEmpty().isInt(),
+    validator.body("expiration").notEmpty().escape(),   // No validator for checking if Date is in ISO format.
```
Comment (wenyngcar) [inline backend/src/middleware/api/router.js:64]: Use schema

File: backend/src/middleware/api/router.js [hunk 2]
```diff
@@ -35,46 +36,69 @@ router.get("/products", async (req, res) => {
...
 // NOTE: Unused
-router.post("/create-variant", async (req, res) => {
-    console.log();
-    console.log("Creating variant with the following information:");
-    console.log(req.body);
-    console.log();
+router.post("/create-variant", 
+    validator.body("name").notEmpty().escape(),
+    validator.body("description").notEmpty().escape(),
+    validator.body("category").notEmpty().escape(),
```
Comment (wenyngcar) [inline backend/src/middleware/api/router.js:88]: Use schema

**Review by wenyngcar — CHANGES_REQUESTED** [review]
Use schema instead of specifying validation in functions.
You can refer to this page: https://express-validator.github.io/docs/guides/schema-validation

**Comment by MiguelCanedo** [comment]
I have replaced the functions with the validation schemas.

I didn't place the schemas into a separate file as I don't think we'll be reusing it.

**Review by wenyngcar — APPROVED** [review]

**Pull request merged**